docs/source/faq.rst

```diff
@@ -172,3 +172,22 @@ paths.  In most cases, the following paths are checked in order:
 Note that if you're using conda, activating an environment sets
 :envvar:`PROJ_LIB` to a resource directory located in that environment.
 
+
+What happend to PROJ.4?
+--------------------------------------------------------------------------------
+
+The first incarnation of PROJ saw the light of day in 1983. Back then it
+was simply known as PROJ. Eventually a new version was released, known
+as PROJ.2 in order to distinguish between the two versions. Later on both
+PROJ.3 and PROJ.4 was released. By the time PROJ.4 was released the
+software had matured enough that a new major version release wasn't an
+immediate necesity. PROJ.4 was around for more than 25 years before it
+again became time for an update. This left the project in a bit of a
+conundrum regarding the name. For the majority of the life-time of the product it was known as PROJ.4, but with the release of version 5 the name
+was no longer aligned with the version number. As a consequence, it was
+decided to decouple the name from the version number and once again simply
+call the software PROJ.
+
+Use of name PROJ.4 is now strictly reserved for describing legacy behaviour
+of the software, e.g. "PROJ.4 strings" as seen in :program:`projinfo`
+output.
```

Comment:
There is also Gerry's "libproj4" effort, that was a fork away from the work that @warmerdam had done in PROJ4 in regards to shifting. I don't know if we need to add that here, but it is a little bit more lineage.

Reply:
I think that is material for the history section, if we ever get round to writing that (#1539)